opm/autodiff/BlackoilWellModel_impl.hpp

```diff
@@ -121,39 +154,110 @@ namespace Opm {
         // calculate the efficiency factors for each well
         calculateEfficiencyFactors();
 
+        const Grid& grid = ebosSimulator_.vanguard().grid();
+
         if (has_polymer_)
         {
             if (PolymerModule::hasPlyshlog()) {
                 computeRepRadiusPerfLength(grid);
             }
         }
 
-        // update VFP properties
```

Comment:
Is the following is done somewhere else?

Reply:
It is done in beginReportStep(), no need to redo this every timestep